server/shared/src/main/scala/org/http4s/server/middleware/ResponseTiming.scala

```diff
@@ -43,12 +44,9 @@ object ResponseTiming {
       headerName: CIString = ci"X-Response-Time",
   )(implicit F: Sync[F], clock: Clock[F]): HttpApp[F] =
```

Comment:
`Sync` extends `Clock`, so including both is ambiguous.

Reply:
I can do this refactoring too 👍🏻

Reply:
But wait. What's about MiMa?

Reply:
Come on, you know the drill 😛 

```scala
@deprecated
def apply[F[_]](
  http: HttpApp[F],
  timeUnit: TimeUnit = MILLISECONDS,
  headerName: CIString = ci"X-Response-Time",
  F: Sync[F],
  clock: Clock[F]
) = ???

Reply:
mehhhhh!

Reply:
Actually, why do we even need `Sync` here?

Reply:
Functor + Clock should work well if you wanna get purity at max 😄

Reply:
Have we beaten all the dragons?